cmake/BuildLibSanitizers.cmake

```diff
@@ -23,36 +23,65 @@ macro(GEN_BUILD_TARGET name libsanitizer_path libfuzzer_path
 
```

Comment:
Your commit adds macOS support, please add an entry to changelog to the section `Added`.